code/frameworks/nextjs/src/export-mocks/headers/index.ts

```diff
@@ -0,0 +1,13 @@
+import { fn } from '@storybook/test';
+import * as originalHeaders from 'next/headers.actual';
+
+// re-exports of the actual module
+export * from 'next/headers.actual';
+
+// mock utilities/overrides (as of Next v14.2.0)
+export { headers } from './headers';
```

Comment:
So even though these collide, the latter will win because it's last? TIL. I thought JS would just crash.

Reply:
Yes, they should overwrite the exports!